src/components/form/input-field.tsx

```
@@ -0,0 +1,95 @@
+type InputProps = InputHTMLAttributes<HTMLInputElement>;
+
+interface InputFieldProps<T extends z.ZodType> {
+  control: Control<z.infer<T>>;
+  name: Path<TypeOf<T>>;
+  label?: string;
+  icon?: ForwardRefExoticComponent<
+    Omit<LucideProps, "ref"> & RefAttributes<SVGSVGElement>
+  >;
+  description?: string | ReactNode;
+  placeholder?: string;
+  onChange?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)